src/implementations/roaring.rs

```diff
@@ -67,67 +66,44 @@ mod tests {
 	}
 
 	#[test]
-	fn test_roaring_serialization_benchmark() {
+	fn test_roaring_serialization_space_benchmark() {
 		let mut val = RoaringTreemap::new();
-		for i in 0..10000 {
+		for i in 0..1_000_000 {
```

Comment:
If we need to keep this benchmark here do we still need to increase the sample size? This will slow down the tests quite a bit, especially with `cargo hack` in the mix.

Reply:
Now that it moves to the benchmark we can even increase it as it runs with the optimised binary